config.toml

```diff
@@ -8,7 +8,194 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 themesDir = "node_modules/"
 enableRobotsTXT = true
 
-[Params]
+[languages]
+  [languages.en]
+  [languages.cn]
+
+[languages.cn.Params]
```

Comment:
I would suggest dividing config into parts, so they can be localized by name. (menus.en.toml, menus.cn.toml etc.)
This way they can be more easily maintained, because if we add one or two more languages in future it would be impossible to maintain one file.

Info: https://gohugo.io/getting-started/configuration/ (files can be localized to become language specific.)

Reply:
I can take care of that as a separate issue, as I've done this with our base theme to support other languages. I'd prefer to get this unblocked for the moment so that we can get the base product working rather than getting the perfect configuration set up